Homework1/ISBN/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Linq;
+
+namespace ISBN
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // Entry messages
+            Console.WriteLine("Hello. This is an ISBN calculator =) \nEnter a nine digit string. ");
+            Console.WriteLine("\n\nNumbers: ");
+            var inputStr = Console.ReadLine();
+
+            if (inputStr.Length == 9 && inputStr.All(char.IsDigit))
+            {
+                int sumOfNineDigits = 0;
+
+                // Each digit checking and summing
+                foreach (var digit in inputStr)
+                {
+                    sumOfNineDigits += int.Parse(digit.ToString());
+                }
+
+                int controlNum = 0;
+
+                // Iterating through 10 numbers to find the divisible of 11.
+                // The closest divisibles could be only in range of 10 numbers (from 81 - sum of
+                // maximum possible number of user's input)
+                for (int i = 0; i <= 10; i++)
+                {
+                    if ((sumOfNineDigits + i) % 11 == 0)
```

Comment:
The alg does not look right. It should not just sum up numbers but multiply them on [10..1] sequence. 
Test your app with https://www.isbn-check.com/

Input : 0112112425
```
0112112425 = 10*0 + 9*1 + 8*1 + 7*2 + 6*1 +
           5*1 + 4*1 + 3*4 + 2*2 + 1*5 = 71
```
Since 71 is not divisible by 11, given number
is not a valid ISBN.